src/test/rustdoc-js-std/alias-2.js

```diff
@@ -4,6 +4,8 @@ const QUERY = '+';
 
 const EXPECTED = {
     'others': [
+        { 'path': 'core::ops', 'name': 'AddAssign' },
+        { 'path': 'core::ops', 'name': 'Add' },
         { 'path': 'std::ops', 'name': 'AddAssign' },
         { 'path': 'std::ops', 'name': 'Add' },
```

Comment:
This isn't sorted correctly. The results for the current crate (`std`) should be listed first which is what happens for other search results.

Reply:
Ok, I'll make the current crate items go first.

Reply:
I enforced it in this test as well that `std` aliases were the first ones.